tests/Unit/Http/Middleware/ResolveLocationTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tipoff\Locations\Tests\Unit\Http\Middleware;
+
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Illuminate\Support\Facades\Route;
+use Tipoff\Locations\Models\Location;
+use Tipoff\Locations\Models\Market;
+use Tipoff\Locations\Services\LocationRouter;
+use Tipoff\Locations\Tests\TestCase;
+
+class ResolveLocationTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    /** @test */
+    public function single_market_single_location()
+    {
+        $this->logToStderr();
+        Route::getLocation('test', 'test', function () {
+            return LocationRouter::build('test');
+        });
```

Comment:
@drewroberts this is an example of how other packages should register location dependent routes.  Other than passing route name as first parameter, syntax is same as `Route::get(...)` - this test version is using an inline function but all variations of controller definition remain supported as well.

Reply:
This is great! Thank you!